src/extensionFunctions.ts

```diff
@@ -898,3 +903,54 @@ export async function copyFileTextToClipboard(uri: vscode.Uri) : Promise<void> {
 		}
 	}
 } 
+
+export async function pasteClipboardToActiveEditor() : Promise<void> {
+	let currentEditor : vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+	if (!currentEditor) {
+		await vscode.commands.executeCommand('workbench.action.openPreviousRecentlyUsedEditor');
+		currentEditor = vscode.window.activeTextEditor;
+	}
+	if (currentEditor) {
+		const textFromClipboard = await vscode.env.clipboard.readText();
+		if (!textFromClipboard) {
+			await vscode.window.showWarningMessage(`No text found on clipboard`);
+			return;
+		}
+		await currentEditor.insertSnippet(new vscode.SnippetString(textFromClipboard));
+		return;
+	}
+	await vscode.window.showWarningMessage(`No active text editor found to paste from clipboard`);
```

Comment:
to avoid having return statements spreads on several lines:
```suggestion
	if (currentEditor) {
		const textFromClipboard = await vscode.env.clipboard.readText();
		if (textFromClipboard) {
			await currentEditor.insertSnippet(new vscode.SnippetString(textFromClipboard));	
		} else {
			await vscode.window.showWarningMessage(`No text found on clipboard`);
		}
	} else {
		await vscode.window.showWarningMessage(`No active text editor found to paste from clipboard`);
	}
```